### PR TITLE
Parallel tests

### PR DESCRIPTION
**Pull request by Paul-Hess**
this seems like an easy win, was able to get it running while debugging other stuff in the background. For general unit testing it halves the total run time 👍

### CODE REVIEW

**Comment by lkemmerer** [comment]
Sweet! Does it pass in CI and everything?

File: src/test/resources/junit-platform.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+junit.jupiter.execution.parallel.enabled=TRUE
```
Comment (Paul-Hess) [inline src/test/resources/junit-platform.properties]: This makes it to run tests in parallel on N threads
https://junit.org/junit5/docs/current/user-guide/#writing-tests-parallel-execution

File: src/test/java/com/flightstats/hub/webhook/WebhookStateReaperTest.java [hunk 2]
```diff
@@ -30,6 +32,7 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
+@Execution(ExecutionMode.SAME_THREAD)
```
Comment (Paul-Hess) [inline src/test/java/com/flightstats/hub/webhook/WebhookStateReaperTest.java]: A few of our tests run over each other's state if run in parallel, i.e. trying to create the same node in zookeeper at the same time as another. This execution mode overrides the default parallel run setting.

**Comment by Paul-Hess** [comment]
About to run it in Jenkins, have only passed it locally so far

**Comment by Paul-Hess** [comment]
> Sweet! Does it pass in CI and everything?

>> About to run it in Jenkins, have only passed it locally so far

Passed and shaved 1+ minute off the average build time

File: src/test/java/com/flightstats/hub/webhook/WebhookStateReaperTest.java (hunk 2 above)
Comment (dkleckler) [reply src/test/java/com/flightstats/hub/webhook/WebhookStateReaperTest.java]: Cool.  I've definitely run across some classes that can't be internally parallelized.  Nice feature.

File: src/test/resources/junit-platform.properties (hunk 1 above)
Comment (dkleckler) [reply src/test/resources/junit-platform.properties]: Nice. This way we have to explicitly flag our tests that require SAME_THREAD...which probably marks them as good refactoring opportunities.

**Pull request merged**